vrx_gazebo/src/vrx_gazebo/generate_worlds.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Are these changes backwards compatible? If so, I think `python` in the shebang will make it compatible with Python2.

Reply:
@caguero Verifying that now...

Reply:
In the default setup of Focal the following give an error:
```
$ python
bash: python: command not found
$ /usr/bin/env python
env: ‘python’: No such file or directory
$ /usr/bin/python
env: ‘python’: No such file or directory
```
In other words, there is no longer a versionless "python" alias that points to anything. You have to either set up the alias or explicitly ask for a version of python that's installed on your system. Since we have updated requirements to Focal, 
python3 is installed by default, so I agree with updating to point to this.

Users still working on Bionic should have no problem as long as they have Python 3 installed. If someone is really trying to avoid installing Python 3 and use Python 2 only, they would have to do some work, but in my opinion we should wait to address this until we understand the use case. For most people it seems pretty reasonable to just add Python 3 as a dependency.

Reply:
@caguero OK to merge this?

Reply:
Thanks for the verification, it looks good to me as it is then.